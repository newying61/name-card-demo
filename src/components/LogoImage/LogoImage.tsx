import React from 'react';
import logo from './logo.svg';

const LogoImage: React.FC = React.memo(() => {
  return (
    <img src={logo} alt="logo" width="80" height="80" role="presentation" aria-hidden="true" />
  );
});

export default LogoImage;
