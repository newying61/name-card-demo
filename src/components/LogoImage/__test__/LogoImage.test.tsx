import React from 'react';
import { render, cleanup } from '@testing-library/react';
import LogoImage from '../';

afterEach(cleanup);

describe('AppNav', () => {
    it('should render without crash', () => {
      const { container } = render(
        <LogoImage />
      );

      expect(container.firstChild).toMatchSnapshot();
  });
});