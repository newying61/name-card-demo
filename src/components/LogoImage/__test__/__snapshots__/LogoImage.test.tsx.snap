// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNav should render without crash 1`] = `
<img
  alt="logo"
  aria-hidden="true"
  height="80"
  role="presentation"
  src="logo.svg"
  width="80"
/>
`;
